src/rules.html

```diff
@@ -180,9 +180,17 @@ <h2>3. Griefing</h2>
       </li>
       <li id="3.3">
         <b>3.3	Town claims</b><br/>
+        Creating claim arms extending from towns is not allowed.
+        These are protrusions which are often used to block the expansion of other towns or just artificially expand the area a town controls without the need to claim all of it.<br>
+        Claim arms that follow a certain strip of land surrounded by water like a peninsula or series of islands are allowed however as no one would be claiming around it anyway.<br>
+        <br>
         Towns are not allowed to have areas of unclaimed chunks inside of their borders as this can substantially increase the size of a town without them actually having to increase their claim limit legitimately.<br>
         <br>
-        Surrounding other towns with claims is also not allowed, about 3/4 of the town has to be surrounded for it to count as illegal. The only exception to this is when a town is created in an area where it is already surrounded. 
+        Towns which are clearly created for no other reason than to block an existing town will also be removed.
+        This applies to situations such as creating a town in the sea on the border of another town as there would be no reason to claim there other than to block the existing town.
+        It does not apply to someone just placing a town close to an existing one if there is not much room to claim in the area.
```

Comment:
Just to clarify does this mean that a town without the intention to claimblock thats next to another town isnt allowed if there is room in the area? It never says it isnt allowed but its saying its not excepted?